guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -571,3 +665,93 @@ done
 ----
 # hammer product list --organization $ORG --sync-plan $SYNC_PLAN
 ----
+
+[[Importing_Content-Limiting_Synchronization_Speed]]
+
+=== Limiting Synchronization Concurrency
+
+By default each Repository Synchronization job will fetch up to 10 files at a time.  This can be adjusted on a per repository basis.
```

Comment:
```suggestion
By default each Repository Synchronization job can fetch up to 10 files at a time.  This can be adjusted on a per repository basis.
```